sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/UniqueKeyPolicy.java

```diff
@@ -34,7 +34,7 @@ public UniqueKeyPolicy() {
      *
      * @return the unique keys.
      */
-    public Collection<UniqueKey> uniqueKeys() {
+    public List<UniqueKey> setUniqueKeys() {
```

Comment:
Isn't it supposed to be  getUniqueKeys?

Reply:
correct. intelliji refactoring messed up. thanks for catching will fix.

Reply:
addressed.